docs/UserGuide.md

```diff
@@ -144,80 +160,113 @@ This would display the following in a pop-up window
+
+![View STATS Display](images/view/view_stats.jpg) _The display showing the stats of *ALL* the students._
 
 ### View full student list on the home page: `view`
 This would display all students currently added to the list. Each student's contact information and subject information will be displayed with this command.
-Format: `view -all`
+Format: `view -all` <br>
+
+After entering the command, the interface will update as shown below:
+
+![View ALL Display](images/view/view_all.jpg) _The display showing *ALL* the students._
 
 ### View student particular by name: `view`
 This will display students whose name matches any single word in the `{NAME}` search field.
 * i.e. `view -name alex tan` will retrieve any student with the word `alex` or `tan` in their name. `Alex Yeo` and `James Tan` are both valid students to display in this case.
 Format: `view -name {NAME}`
 Examples:
-* `view -name Xiao Ming` would display the student particular of Xiao Ming if existed
+* `view -name Benson Meier` would display the student particular of Benson Meier if existed or any student with the word `Benson` or `Meier` in their name. <br>
+
+After entering the command, the interface will update as shown below:
+
+![View name Display](images/view/view_name.jpg) _The display showing *ALL* the students with the word `Benson` or `Meier` in their name._
+
 
 ### View student particular by id: `view`
 This will display a specific student by searching for their id, this also pops up the lesson logs of the student.
 Format: `view -id {ID}`
 
 Examples:
-* `view -id 88888` would display the student particular for the student whose id is 88888 if it exists.
+* `view -id 000003` would display the student particular for the student whose ID is #000003 if it exists. <br>
+
+After entering the command, the interface will update as shown below:
+
+![View ID Display](images/view/view_id.jpg) _The display showing the student with the ID #000003._
 
 ## Enhanced Payment Management
 
 TuteeTally's payment management commands are designed to streamline the financial interactions between tutors and students, ensuring accuracy and transparency. Below are detailed explanations of each command's purpose and benefits:
 
-### Adding Payments: `addpayment`
+### Adding a Payment: `addpayment`
 
 Record new payments easily with the `addpayment` command, keeping track of what each student owes. This command simplifies the maintenance of financial records, ensuring you never overlook an outstanding payment.
 
-**Format:** `addpayment -id {ID} -payment {AMOUNT}`
-**Example:** `addpayment -id 000001 -payment 150` or `addpayment -id 1 -payment 150` would add a payment of 150 to the student whose ID is '#000001' if it exists. <br>
+Format: `addpayment -id {ID} -payment {AMOUNT}`
+
+Example: 
+* `addpayment -id 000001 -payment 150` or `addpayment -id 1 -payment 150` would add a payment of 150 to the student whose ID is '#000001' if it exists. <br>
```

Comment:
minor formatting, do we want to represent all ids this way '#000001'? some are without quotations

Reply:
removed all quotations